mongodb/mongod.conf

```diff
@@ -0,0 +1,5 @@
+storage:
+  dbPath: ./mongodb/db
+net:
+  bindIp: 127.0.0.1, ::1
+  ipv6: true
```

Comment:
Does this affect team members who do not have a modem with ipv6?

Reply:
Do you have a modem with ipv6?